ChromecastSampleApp/ChromecastSampleApp/Players/PlayerViewController.m

```diff
@@ -87,7 +87,9 @@ - (void)viewDidLoad {
   // Init the castManager in the ooyalaPlayer
   [self.ooyalaPlayer initCastManager:self.castManager];
   [self.ooyalaPlayer setEmbedCode:self.embedCode];
-  [self.ooyalaPlayer play];
+  if( self.mediaInfo.startPlaying ) {
```

Comment:
I don't follow. We'd still need to check if it was paused or not, no? Somewhere. I guess it could just be in the PlayerViewController insetad of the CLVC, true, but still seems to me to need to be checked?